common/pixman-region.c

```diff
@@ -1809,7 +1809,7 @@ validate (region_type_t * badreg)
         FREE_DATA (&ri[i].reg);
 
     if (ri != stack_regions)
-        free (ri);
+        free (ri); /* cppcheck-suppress autovarInvalidDeallocation */
```

Comment:
Please don't hesitate to apply the coding standard when you touch lines that violate the standard.

Reply:
OK - I'll do so. The reason I left this one was because of the history of the file.